loglist2/logfilter_test.go

```diff
@@ -130,3 +131,59 @@ func TestRootCompatible(t *testing.T) {
 		})
 	}
 }
+
+// stripErr is helper func for wrapping time.Parse
+func stripErr(t time.Time, _ error) time.Time {
+	return t
+}
+
+func TestTemporalCompatible(t *testing.T) {
+	cert, _ := x509util.CertificateFromPEM([]byte(testdata.TestPreCertPEM))
+
+	tests := []struct {
+		name      string
+		in        LogList
+		cert      *x509.Certificate
+		notBefore time.Time
+		want      LogList
+	}{
+		{
+			name:      "AllFit",
+			in:        sampleLogList,
+			cert:      cert,
+			notBefore: stripErr(time.Parse(time.UnixDate, "Sat Nov 8 11:06:00 PST 2014")),
+			want:      subLogList(map[string]bool{"https://ct.googleapis.com/aviator/": true, "https://log.bob.io": true, "https://ct.googleapis.com/icarus/": true, "https://ct.googleapis.com/racketeer/": true, "https://ct.googleapis.com/rocketeer/": true}),
+		},
+		{
+			name:      "OperatorOff",
```

Comment:
It wasn't immediately obvious to me what "OperatorOff" meant. I think it means that log.bob.io was retired by this `notBefore` time? Could you make the test names a bit more verbose please?

Reply:
Done